File: src/main/java/mods/eln/sim/electrical/mna/state/VoltageStateLineReady.java
package mods.eln.sim.electrical.mna.state;

public class VoltageStateLineReady extends VoltageState {

    boolean canBeSimplifiedByLine = false;

    public void setCanBeSimplifiedByLine(boolean v) {
        this.canBeSimplifiedByLine = v;
    }

    @Override
    public boolean canBeSimplifiedByLine() {
        return canBeSimplifiedByLine;
    }
}

File: src/main/java/mods/eln/sim/mna/state/VoltageStateLineReady.kt
package mods.eln.sim.mna.state

open class VoltageStateLineReady : VoltageState() {

    init {
        name = "VoltageStateLineReady"
    }

    internal var canBeSimplifiedByLine = false

    fun setCanBeSimplifiedByLine(v: Boolean) {
        this.canBeSimplifiedByLine = v
    }

    override fun canBeSimplifiedByLine(): Boolean {
        return canBeSimplifiedByLine
    }
}
